### PR TITLE
Update plugin for linting and coveralls and new validation

### PR DESCRIPTION
**Pull request by rachelnicole**
Added updated eslint / eslint ava / testing versions and new validation scripts.

### CODE REVIEW

**Comment by Snugug** [comment]
The plugin's settings still includes `domains` and `regex`. Please remove them.

File: lib/validation.js [hunk 1]
```diff
@@ -1,36 +1,27 @@
...
-  if (!Array.isArray(domains)) {
-    domains = [String(domains)];
-  }
+const isEmail = require('validator/lib/isEmail');
 
-  if (!settings.target.regex.test(input.target.value)) {
-    return 'Not a valid e-mail address.';
-  }
+module.exports = function emailValidation(input) {
+  isEmail(input.target.value);
```
Comment (scottnath) [inline lib/validation.js]: is this line extraneous?

Comment (rachelnicole) [inline lib/validation.js]: It is, I'll clean it up.

**Comment by scottnath** [comment]
👍 
@Snugug ?

**Pull request merged**